### DISCUSSION TITLE
New to Django, is there an easy way to use this jazzmin style for other templates

### DISCUSSION THREAD

**Discussion by observer117** [farridav/django-jazzmin#343] [Q&A] [answered]
Hello,

First of all, thank you for making this awesome looking package. I am a newcomer to Django programming (and python too).
I am thinking whether there is an easy way to implement this jazzmin style for other views and templates I created. For example, the Model ListView and DetailView out of Django box. Much appreciated!

Best,

**Comment by shaishguni** [comment, marked as answer]
Yeah, there is if you just want to just custom the admin just download the static and templates and keep it as it is in your project if you already have one template/static dir then you can just paste the internal dir of it and it's done the same to the static of its already made.

**Reply by PacificGilly** [reply]
Yeah this is sort of mentioned in the docs. Thanks for this solution and the question, both.

https://django-jazzmin.readthedocs.io/configuration/#adding-extra-actions-to-models-form-view

**Comment by observer117** [comment]
Thank you very much!

**Reply by shaishguni** [reply]
you are welcomed and is the question answered if yes please mark it answered :)

**Comment by PacificGilly** [comment]
Closing as this discussion has an answer. Feel free to reopen/comment if needed.
